sandbox_storage/scripts/populate.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+
+# Can't populate S3 yet
+# So, just take example files and hard links right now
+
+"""
+Provides a script function to populate a database
+"""
+
+from datetime import datetime
```

Comment:
```suggestion
from datetime import datetime
from pathlib import Path
```
needed for the next suggestion